README.md

```diff
@@ -1 +1,61 @@
-# chrony
\ No newline at end of file
+[![Build Status](https://travis-ci.org/oasis-roles/chrony.svg?branch=master)](https://travis-ci.org/oasis-roles/chrony)
+
+ROLE NAME
+===========
+
+Basic description for chrony
+
+Requirements
+------------
+
+Ansible 2.4 or higher
+
+Red Hat Enterprise Linux 7 or equivalent
+
+Valid Red Hat Subscriptions
+
+Role Variables
+--------------
+
+Currently the following variables are supported:
+
+### General
+
+* `chrony_package` - The package name to install on remote hosts (set to `chrony` by default)
+* `chrony_become_user` - The user to sign into when running commands on remote hosts (set to `root` by default)
+* `chrony_conf_template_path` - Path to a jinja2 template for generating the chrony configuration file (set to `chrony.conf` by default)
+* `chrony_conf_path` - Path in which to store the generated chrony configuration file on the remote hosts (set to `/etc/chrony.conf` by default)
```

Comment:
Same with this - is this really something we want the user to be able to mangle? Is this path different between RHEL7 and current Fedora, for instance?